containerfs/bin/tc-db-fetch

```diff
@@ -9,13 +9,10 @@ TC_RELEASES_URL="https://github.com/TrinityCore/TrinityCore/releases"
 DB_VERSION=$(grep -o -E '(TDB_full[^"]+)' /hostfs/tc-server/source/revision_data.h.in.cmake)
 BUILD_NUM=$(node -p "'"$DB_VERSION"'.match(/(335\.[0-9]+)/)[1]")
 
-cd /hostfs/tc-db/
+cd /hostfs/tc-wd/
```

Comment:
@josefalcon this was responsible for the grants.sql turning into a directory. `cd` to the wrong directory, `rm *.sql`, then docker, rather than erroring, creates a folder as its default when mounting a bind volume.

Reply:
how'd you find this?!

Reply:
luckily it was failing on CI! Once i figured out how to get the binaries cached, the cycle time was short enough to actually figure out the problem.

Also, I did:

```
ls -lahR .
```

and just dumped the entire file tree between steps to see when it changed.

Reply:
Also, I was googling for solutions when I found that if a mount location does not exist, docker creates a folder as default rather than erroring. So it was being deleted by this script accidentally, then when starting the container a folder was created since there was no file by that name.